spec/observables/dom/ajax-spec.ts

```diff
@@ -1173,6 +1170,9 @@ class MockXMLHttpRequest {
   constructor() {
     MockXMLHttpRequest.recentRequest = this;
     MockXMLHttpRequest.requests.push(this);
+    if (MockXMLHttpRequest.noResponseProp) {
+      delete this['response'];
+    }
```

Comment:
Above here is _part_ of the magic for removing the `response` property from the XHR to simulate older browsers.